src/App.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import getAmiibos from './API';
+import Navbar from './Components/navBar';
+import Home from './Pages/Home';
+import Details from './Components/nintendoDetails';
+
+function App() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getAmiibos());
+  }, []);
+
+  return (
+    <BrowserRouter>
+      <Navbar />
+      <Routes>
+        <Route path="/details/:id" element={<Details />} />
+        <Route index element={<Home />} />
+      </Routes>
+    </BrowserRouter>
+  );
+}
+
+export default App;
```

Comment:
- [ ] Good job so far cause your application is very well implemented :heavy_check_mark:  But you have to work on your app design, it is a bit different to the original design :cry:  Please I would like to recommend you follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) by using the same layout, position to make it similar to the original design. :+1: 
Please check the screenshots to see the differences  below :arrow_down: 

#### Your design 

![Screenshot from 2022-08-31 17-25-28](https://user-images.githubusercontent.com/6615394/187742518-2589b622-3bd0-4429-ad6a-2183d6ac8288.png)
![Screenshot from 2022-08-31 17-25-35](https://user-images.githubusercontent.com/6615394/187742509-fde2be1d-f597-4f4a-acea-c913f27e10b7.png)

#### Original design
<img width="421" alt="metrics_app" src="https://user-images.githubusercontent.com/6615394/187742519-59d05e87-670a-4fd1-a83b-dfb4e3cf6644.png">